pytomorrowio/pytomorrowio.py

```diff
@@ -97,19 +98,17 @@ def __init__(
             "units": self.unit_system,
         }
         self._headers = {**HEADERS, "apikey": self._apikey}
-        self._rate_limits: Optional[Dict[str, Any]] = None
+        self._rate_limits: CIMultiDict = CIMultiDict()
```

Comment:
`CIMultiDict` is case-insensitive dictionary that allows for duplicate keys. Since HTTP headers keys are not case-sensitive, we need to allow for the same when looking up values.